buyer-api/src/routes/notarizationResult.js

```diff
@@ -58,12 +29,47 @@ const router = Router();
  *         description: When some field is missing or incorrect.
  *       500:
  *         description: When the fetch failed.
+ *
+ * definitions:
+ *   NotarizationResult:
+ *     type: object
+ *     required:
+ *       - orderId
+ *       - notaryAddress
+ *       - notarizationPercentage
+ *       - notarizationFee
+ *       - payDataHash
+ *       - lock
+ *       - sellers
+ *     properties:
+ *       orderId:
+ *         type: number
+ *         description: The unique identifier for the order.
+ *       notaryAddress:
+ *         type: string
+ *         description: Notary's ethereum address.
+ *       notarizationPercentage:
+ *         type: number
+ *         description: Percentage of orders that have been notarized.
+ *       notarizationFee:
+ *         type: number
+ *         description: Flat one-time payment for the notarization services.
+ *       payDataHash:
+ *         type: string
+ *         description: Seller ids array hash.
+ *       lock:
+ *         type: string
+ *         description: The lock for the BatPay.Transfer.
+ *       sellers:
+ *         type: array
+ *         description: List of sellers send to notary.
+ *         items:
+ *           type: string
```

Comment:
This is expected to be an object
```suggestion
 *           type: object
```
I wouldn't mind putting the exact scheme rigth now.

Reply:
F&$*, u you are right, gonna check that out.